build-tools/packages/build-cli/src/genMonoRepoPackageJson.ts

```diff
@@ -0,0 +1,216 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import {
+    MonoRepo,
```

Comment:
I was thinking of having this inside a helper folder or some other name, wherein we can eventually move all the functions from build-tools to new build-cli. 

@tylerbutler  -- any thoughts around how we plan to move everything to new build-cli?

Reply:
You mean like import into a module in the build-cli project, then re-export? Then within build-cli, we'd use the re-exported functions? Or did you mean something different?